src/test/resources/feature/BDDscenarioForWishList.feature

```diff
@@ -0,0 +1,46 @@
+@WishList
+Feature: Wish List;
+
+  Background:
+    Given user is on http://automationpractice.com/ page
```

Comment:
Here you cand write something like: user is on home page. Is no need to specify direct link, you can handle this in StepDef

Reply:
done